### PR TITLE
Added readme and changelog for the @yoast/schema-blocks package

### PR DESCRIPTION
**Pull request by Xyfi**
## Summary

This PR can be summarized in the following changelog entry:

*

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

*

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [ ] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #

### CODE REVIEW

File: packages/schema-blocks/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+This changelog is according to [Keep a Changelog](http://keepachangelog.com).
+
+All notable changes to this project will be documented in this file.
+We follow [Semantic Versioning](http://semver.org/).
+
+## 1.1.0 Januari 11th, 2021
```
Comment (andizer) [inline packages/schema-blocks/CHANGELOG.md:8]: Is this date right? Because it lies in the past.

```suggestion
## 1.1.0 January 11th, 2021
```

**Review by andizer — CHANGES_REQUESTED** [review]
One question about the date  that is given in the changelog.

File: packages/schema-blocks/CHANGELOG.md (hunk 1 above)
Comment (increddibelly) [reply packages/schema-blocks/CHANGELOG.md]: 1.10 was released without changelog. we're retrofitting that now :-)

Comment (hansjovis) [reply packages/schema-blocks/CHANGELOG.md]: 1.1.0 has been released on January 11th, so this date is right.

[According to NPM we released four 'stable' versions: 1.0.0, 1.1.0, 1.1.1 and 1.2.0.](https://www.npmjs.com/package/@yoast/schema-blocks) 1.3.0 is now in the RC process. I would expect changelog items for those versions as well.

File: packages/schema-blocks/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+This changelog is according to [Keep a Changelog](http://keepachangelog.com).
+
+All notable changes to this project will be documented in this file.
+We follow [Semantic Versioning](http://semver.org/).
+
+## 1.1.0 January 11th, 2021
+### Enhancements
+* Creates a new package for the Schema blocks API that is used for our schema generation.
```
Comment (hansjovis) [inline packages/schema-blocks/CHANGELOG.md:10]: [According to NPM we released four 'stable' versions: 1.0.0, 1.1.0, 1.1.1 and 1.2.0. 1.3.0 is now in the RC process.](https://www.npmjs.com/package/@yoast/schema-blocks) I would expect changelog items for those versions as well.

**Review by hansjovis — CHANGES_REQUESTED** [review]

**Pull request merged**